wom/src/main/scala/wdl4s/wdl/WdlExpression.scala

```diff
@@ -186,6 +192,43 @@ case class WdlExpression(ast: AstNode) extends WdlValue {
   def variableReferences: Iterable[VariableReference] = AstTools.findVariableReferences(ast)
 }
 
+/**
+  *
+  * @param wdlExpression The wrapped WdlExpression.
+  * @param from The Scope in which the WdlExpression is found, needed to adjust member access expressions located in
+  *             conditionals (wrapped in optionals) or scatters (wrapped in arrays).
+  */
+final case class WdlWomExpression(wdlExpression: WdlExpression, from: Option[Scope]) extends WomExpression {
+
+  override def inputs: Set[String] = wdlExpression.variableReferences map { _.fullVariableReferenceString } toSet
+
+  override def evaluateValue(variableValues: Map[String, WdlValue], ioFunctionSet: IoFunctionSet): ErrorOr[WdlValue] = {
+    lazy val wdlFunctions = new WdlStandardLibraryFunctions {
+      override def readFile(path: String): String = Await.result(ioFunctionSet.readFile(path), Duration.Inf)
+
+      override def writeFile(path: String, content: String): Try[WdlFile] = Try(Await.result(ioFunctionSet.writeFile(path, content), Duration.Inf))
+
+      override def stdout(params: Seq[Try[WdlValue]]): Try[WdlFile] = ???
+
+      override def stderr(params: Seq[Try[WdlValue]]): Try[WdlFile] = ???
+
+      override def glob(path: String, pattern: String): Seq[String] = ???
+
+      override def size(params: Seq[Try[WdlValue]]): Try[WdlFloat] = ???
```

Comment:
I'm not familiar enough with what's going on here but are these todo's or are they purposefully unimplemented?  If so perhaps a comment to help the uninitiated?